app/controllers/users/phone_setup_controller.rb

```diff
@@ -11,7 +11,10 @@ class PhoneSetupController < ApplicationController
     helper_method :in_multi_mfa_selection_flow?
 
     def index
-      @new_phone_form = NewPhoneForm.new(current_user)
+      @new_phone_form = NewPhoneForm.new(
+        current_user,
+        setup_voice_preference: setup_voice_preference?,
```

Comment:
What does `setup_voice_preference` end up being used for, practically speaking?

Reply:
See the first screen in this [Figma](https://www.figma.com/file/LlhYpHhdhewSTFUqKUfsXz/LG-8237-Design-displayed-this-is-a-landline-message?node-id=0%3A1&t=EvHbmUBn8PusWDDZ-0).

When the user is taken back to the setup page, under conditions where we believe they have a landline and should therefore not receive a text, we select the option `phone call` for them as a convenience via a param.

Reply:
I will share a video of the behavior in the private channel.

Reply:
Ah, I see now! I totally missed that as part of the ticket behavior. Makes sense to me 👍 

It could be nice to have a feature spec which runs through and checks that the notice is shown and then that "Phone call" is selected when clicking the call to action link.